index.bs

```diff
@@ -866,12 +859,14 @@ method is invoked, the user agent MUST:
 
 1. Let |issuedRequests| be a new [=ordered set=].
 
-1. If there are no [=authenticators=] currently available on this platform, return a {{DOMException}} whose name is
-    "{{NotFoundError}}", and terminate this algorithm.
-
 1. Let |authenticator| be a platform-specific handle whose value identifies an [=authenticator=].
 
-1. For each |authenticator| currently available on this platform, perform the following steps:
+1. For each |authenticator| that becomes available on this platform within the overall lifetime of
```

Comment:
ditto here wrt "Start a timer for adjustedTimeout milliseconds" being done here.